src/runner/porter-command/portPhoneServiceInsertEventCommand.ts

```diff
@@ -100,7 +101,11 @@ class PortPhoneServiceInsertEventCommand implements PorterCommand {
           async (data) =>
             await manager.transaction(async (manager) => {
               try {
-                const { memberNotes, lastMemberNotes, key } = data;
+                const { lastMemberNotes, key } = data;
+                const memberNotes = data.memberNotes.map((note) => ({
+                  ...note,
+                  createdAt: dayjs(note.createdAt).toDate(),
+                }));
```

Comment:
ORM best practice is use ORM instance 

```ts
  const memberNotes = data.memberNotes.map((note) => {
    const memberNote = new MemberNote();
    memberNote.authorId = note?.authorId || '';
    memberNote.metadata = note?.metadata || null;
    memberNote.duration = note?.duration || 0;
    memberNote.memberId = note?.memberId || '';
    memberNote.status = note?.status || '';
    memberNote.type = note?.type || '';
    memberNote.createdAt = dayjs(note?.createdAt).isValid() ? dayjs(note.createdAt).toDate() : new Date();
    return memberNote;
  });
  
// need to update insert function too
  async insertData(data: MemberNote[], entityManager: EntityManager) {
    const memberNoteRepo = entityManager.getRepository(MemberNote);
    return await memberNoteRepo.insert(data);
  }
```

Reply:
Adjusted, thanks for review.

Reply:
Note:
After some discussion, in addition to checking createdAt here, we also need to check and convert the data source in kolable-server. For details, see: https://github.com/urfit-tech/kolable-server/pull/103/files